mattermost/mattermost
# MM-54976 Fix bug where newly created GMs show as having 0 members

#### Summary
Websocket events for updating a preference or list of preferences will load GM's upon new GM open.
Previously loading DM and GM was done when opening only a DM.
Now the logic is separated to load DM for DM event and GM for GM event.

QA test:
1. open 2 instances of the same user. (web + desktop)
2. create a new group message in 1 session
3. verify that the number members in the GM is correct for both sessions.


#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/25067
Jira https://mattermost.atlassian.net/browse/MM-54976


#### Screenshots


#### Release Note

```release-note
Fixed bug where newly created GMs show as having 0 members
```
